apps/settings/js/modules/panel_utils.js

```diff
@@ -82,12 +82,12 @@ define(function(require) {
         fields.forEach(function(input) {
           key = input.dataset.setting;
           switch (input.type) {
-            case 'radio':
+            case 'gaia-radio':
```

Comment:
Settings should no longer use any `input type="radio"` elements as we should be using gaia-radio everywhere now.